frontend/src/app/supportAdmin/UnarchivePatients/UnarchivePatient.test.tsx

```diff
@@ -141,26 +145,22 @@ describe("Unarchive patient", () => {
     await act(
       async () => await userEvent.click(screen.getByText("Clear filters"))
     );
-    expect(screen.getByLabelText("Testing facility *")).toHaveDisplayValue(
-      "- Select -"
-    );
-    expect(screen.getByLabelText("Organization *")).toHaveDisplayValue(
-      "- Select -"
-    );
+    const [facilityInput] = getFacilityComboBoxElements();
+    const [orgInput] = getOrgComboBoxElements();
+
+    expect(orgInput).toHaveValue("");
+    expect(facilityInput).toHaveValue("");
+
     expect(
       screen.getByText(
         "Filter by organization and testing facility to display archived patients."
       )
     ).toBeInTheDocument();
+    expect(screen.getByText("Search")).toBeDisabled();
+
     // show form errors
-    await clickSearch();
```

Comment:
because the UI active / disable state is now dependent on whether the relevant ID's are defined, I deleted this check.